code/init_mooc_nb.py

```diff
@@ -118,7 +119,11 @@ def init_notebook():
     except NameError:
         pass
 
-    print('Populated the namespace with:\n' + ', '.join(__all__))
+    print('Populated the namespace with:\n' +
+        ', '.join(init_mooc_nb) +
+        ', edx_components.'.join(edx_components.__all__) +
+        ', functions.'.join(functions.__all__))
```

Comment:
But this isn't true: we aren't populating namespace with `functions.bla`, rather just `bla`.